app/api/groups/update_group.go

```diff
@@ -48,12 +48,13 @@ type groupUpdateInput struct {
 	FreeAccess bool `json:"free_access" sql:"column:bFreeAccess"`
 	// Duration after the first use of the password it will expire
 	// Nullable
-	// pattern: \d{2}:\d{2}:\d{2}
+	// pattern: ^\d{2}:\d{2}:\d{2}$
```

Comment:
This pattern is not really correct. Values like 1:2:3 should be considered as correct, while values having more than 59 minutes/seconds should not (see https://github.com/France-ioi/AlgoreaBackend/blob/master/app/formdata/validations.go#L11). It is one more reason to add it as a custom type somehow.

Reply:
Actually, it is a doc, that's not very important (it is should show the format, a bit as an example would do). I improved the pattern to avoid rejecting valid cases. 
Yes, of course, custom types would be nice, but if it is not possible in the current go-swagger implem, that does not worth spending hours implementing this.